lib/model_registry.ex

```diff
@@ -7,6 +7,7 @@ defmodule Plenario.ModelRegistry do
     "float" => :float,
     "integer" => :integer,
     "text" => :string,
+    "date" => :date,
```

Comment:
Not sure we need this. The date type in `DataSetField` is only `timestamptz` and I've been mapping that to a `:utc_datetime`.

I've also added the following:

```
"geometry" -> Geo.Geometry
"jsonb" -> map
```